src/UnderlineNav2/styles.ts

```diff
@@ -10,26 +9,16 @@ export const iconWrapStyles = {
   marginRight: 2,
 }
 
-const smallVariantLinkStyles = {
```

Comment:
**Note for reviewers:** These variants were considered in the initial version of the component but it is decided otherwise at the end. Removing them to reduce the clutter and complexity on the component and styles.